base/src/com/google/idea/blaze/base/qsync/QuerySyncAsyncFileListener.java

```diff
@@ -183,7 +193,7 @@ public void afterQuerySync(Project project, BlazeContext context) {
                     return;
                   }
                   if (requester.changePending.get()) {
-                    requester.requestSyncInternal();
+                    requester.requestSyncInternal(ImmutableList.of());
```

Comment:
This won't work. Consider a sequence
1. File 1 is added
2. Sync Starts
3. File 2 is added
4. Sync Finishes
5. `afterQuerySync` is called (this line)

File 2 won't be synced. The `afterQuerySync` is supposed to be executed here nad catch File 2